### DISCUSSION TITLE
Image Display Issue in Written Lessons - Cyfrin Updraft

### DISCUSSION THREAD

**Discussion by LordAdhaar** [Cyfrin/foundry-full-course-cu#2344] [General]
I'm enjoying the Cyfrin Updraft course, especially the written lessons. I've noticed the images aren't displaying in some lessons - only the img tags are visible. Could this be fixed? It would really enhance the reading experience. Thank you!

**Comment by EngrPips** [comment]
Hey @LordAdhaar, Thanks for this contribution. Yeah, they can be fixed. You should use the issue tab of this Github to properly report them so they can be easily traced and fixed.

**Reply by LordAdhaar** [reply]
Okay, my bad

**Reply by LordAdhaar** [reply]
I'll suggest a fix, could you tell me where the code for the video lectures is ?

**Reply by EngrPips** [reply]
here I believe https://github.com/Cyfrin/Updraft
